rag/vector_store.py

```
@@ -222,26 +223,41 @@ def get_cached_documents(
+        kwargs: Dict[str, Any] = {"where": w} if w else {}
+        return collection.query(
             query_embeddings=[query_embedding],
             n_results=n_results,
-            where=where,
             include=["documents", "metadatas"],
+            **kwargs,
         )
+
+    try:
+        results = _run(where)
     except Exception:
```

Ruff (0.16.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)